src/auto_scheduler/measure_record.cc

```diff
@@ -163,8 +163,10 @@ struct Handler<::tvm::auto_scheduler::SearchTaskNode> {
     writer->WriteArrayItem(std::string(data.workload_key));
     writer->WriteArrayItem(data.target->str());
     writer->WriteArrayItem(*data.hardware_params.get());
-    if (data.target_host.defined()) {
-      writer->WriteArrayItem(data.target_host->str());
+    ::tvm::Target target = data.target, target_host = data.target_host;
```

Comment:
nitpick: split into two lines, also perhaps we don't need the prefix "::tvm::"

Reply:
Sure I can split into two lines. In this file specifically, the name space tvm is not used. The prefix is aloso used in other statements as well. Therefore, I'll keep it here for now.